### ISSUE TITLE
`MatClassDocumenter::document_member_section` handles empty `:exclude-members:` incorrectly. 

### ISSUE THREAD

**Issue by joeced** [sphinx-contrib/matlabdomain#189]
As reported in #188, we get this error 

```
AttributeError: '_All' object has no attribute 'copy'
```

if this directive is used

```
.. autoclass:: folder.ClassNanme
    :members: 
    :exclude-members: 
    :no-undoc-members: 

.. autoattribute:: folder.ClassNanme.propname
```

Note, that the if `:exclude-members:` is empty, Sphinx returns a special object that is always empty, not a `set`.
In `MatClassDocumenter::document_member_section` we try copy this object which is not possible. Later we try to add extra excluded members to this object, which is also not possible.

Further, the option `"exclude-members"` should be `exclude_members_option` and not `member_option`.

Edit:
Fixed the above, but now the `autoattribute` directive error trigger this error

```
Exception occurred:
  File "...\matlabdomain\sphinxcontrib\mat_types.py", line 1613, in for_module
    mod = entities_table[modname]
          ~~~~~~~~~~~~~~^^^^^^^^^
KeyError: 'sphinxcontrib.mat_types'
```

**Comment by joeced** [comment]
@rdzman did you see this?

I have a solution ready, but trying to test if it really works.

**Comment by joeced** [comment]
No, I found the issues. I just wanted to let you know

tir. 30. maj 2023 17.05 skrev Ray Zimmerman ***@***.***>:

**Comment by joeced** [comment]
Fixed in 09c3bc9ba69ac104eebdcdbce367e3cdc08bf13e
